### PR TITLE
Upgrade YGGTorrent

### PR DESCRIPTION
**Pull request by anthonyraymond**
Add new domain for YGGTorrent

### CODE REVIEW

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
- v1 definitions are not supported; revert this change
- don't just overwrite the existing URL, move it to legacy links if it's no longer valid at all and then add the new URL I it's place. Alternatively if it's simply a new alternative URL just add it as another option below the existing entry

**Comment by foux** [comment]
> * v1 definitions are not supported; revert this change
> * don't just overwrite the existing URL, move it to legacy links if it's no longer valid at all and then add the new URL I it's place. Alternatively if it's simply a new alternative URL just add it as another option below the existing entry

I agree for the first part, but unless I'm reading it wrong, he already moved the `.re` (which is not valid anymore) to the legacy links, no?

**Comment by bakerboy448** [comment]
> > * v1 definitions are not supported; revert this change
> 
> > * don't just overwrite the existing URL, move it to legacy links if it's no longer valid at all and then add the new URL I it's place. Alternatively if it's simply a new alternative URL just add it as another option below the existing entry
> 
> 
> 
> I agree for the first part, but unless I'm reading it wrong, he already moved the `.re` (which is not valid anymore) to the legacy links, no?

Ah pre-coffee problems and used to the legacy links being appended and not prepended,  can that legacy link be swapped to the bottom as well otherwise it'll be a pain when pulling future jackett updates

**Comment by bakerboy448** [comment]
Fixes https://github.com/Prowlarr/Prowlarr/issues/948#issuecomment-1095339241

**Comment by foux** [comment]
@anthonyraymond are you sure the new URL shouldn't be https://www5.yggtorrent.la ? I think that is the correct one. Did you test it?

File: definitions/v3/yggtorrent.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ type: semi-private
 encoding: UTF-8
 followredirect: true
 links:
-  - https://www3.yggtorrent.re/
+  - https://yggtorrent.la/
```
Comment (bakerboy448) [inline definitions/v3/yggtorrent.yml:10]:
```suggestion
  - https://www5.yggtorrent.la/
```

File: definitions/v2/yggcookie.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ type: semi-private
 encoding: UTF-8
 followredirect: true
 links:
-  - https://www3.yggtorrent.re/
+  - https://yggtorrent.la/
```
Comment (bakerboy448) [inline definitions/v2/yggcookie.yml:10]:
```suggestion
  - https://www5.yggtorrent.la/
```

File: definitions/v2/yggtorrent.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ type: semi-private
 encoding: UTF-8
 followredirect: true
 links:
-  - https://www3.yggtorrent.re/
+  - https://yggtorrent.la/
```
Comment (bakerboy448) [inline definitions/v2/yggtorrent.yml:10]:
```suggestion
  - https://www5.yggtorrent.la/
```

File: definitions/v3/yggcookie.yml [hunk 4]
```diff
@@ -7,7 +7,7 @@ type: semi-private
 encoding: UTF-8
 followredirect: true
 links:
-  - https://www3.yggtorrent.re/
+  - https://yggtorrent.la/
```
Comment (bakerboy448) [inline definitions/v3/yggcookie.yml:10]:
```suggestion
  - https://www5.yggtorrent.la/
```

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
s/b https://www5.yggtorrent.la/

**Comment by anthonyraymond** [comment]
@bakerboy448 Everything should be alright now, and i just squashed the commits

**Comment by foux** [comment]
> @bakerboy448 Everything should be alright now, and i just squashed the commits

Did you checked it with www3? Because on my side it worked only with www5. It might have been a temporary glitch, but you have to be sure (or add several URLs)

**Comment by bakerboy448** [comment]
> > @bakerboy448 Everything should be alright now, and i just squashed the commits
> 
> Did you checked it with www3? Because on my side it worked only with www5. It might have been a temporary glitch, but you have to be sure (or add several URLs)

Jackett also pushed with ww5

File: definitions/v2/yggcookie.yml [hunk 5]
```diff
@@ -7,7 +7,7 @@ type: semi-private
 encoding: UTF-8
 followredirect: true
 links:
-  - https://www3.yggtorrent.re/
+  - https://www3.yggtorrent.la/
```
Comment (bakerboy448) [inline definitions/v2/yggcookie.yml:10]:
```suggestion
  - https://www5.yggtorrent.la/ 
```

File: definitions/v2/yggtorrent.yml [hunk 6]
```diff
@@ -7,7 +7,7 @@ type: semi-private
 encoding: UTF-8
 followredirect: true
 links:
-  - https://www3.yggtorrent.re/
+  - https://www3.yggtorrent.la/
```
Comment (bakerboy448) [inline definitions/v2/yggtorrent.yml:10]:
```suggestion
  - https://www5.yggtorrent.la/ 
```

File: definitions/v3/yggcookie.yml [hunk 7]
```diff
@@ -7,7 +7,7 @@ type: semi-private
 encoding: UTF-8
 followredirect: true
 links:
-  - https://www3.yggtorrent.re/
+  - https://www3.yggtorrent.la/
```
Comment (bakerboy448) [inline definitions/v3/yggcookie.yml:10]:
```suggestion
  - https://www5.yggtorrent.la/
```

File: definitions/v3/yggtorrent.yml [hunk 8]
```diff
@@ -7,7 +7,7 @@ type: semi-private
 encoding: UTF-8
 followredirect: true
 links:
-  - https://www3.yggtorrent.re/
+  - https://www3.yggtorrent.la/
```
Comment (bakerboy448) [inline definitions/v3/yggtorrent.yml:10]:
```suggestion
  - https://www5.yggtorrent.la/
```

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

**Comment by anthonyraymond** [comment]
@foux You were right, how did you guessed that it was www5? The website Q&A tells that it's www3

@bakerboy448 done

File: definitions/v2/yggtorrent.yml [hunk 9]
```diff
@@ -7,7 +7,7 @@ type: semi-private
 encoding: UTF-8
 followredirect: true
 links:
-  - https://www3.yggtorrent.re/
+  - https://www5.yggtorrent.la/
```
Comment (bakerboy448) [inline definitions/v2/yggtorrent.yml:10]:
```suggestion
  - https://www5.yggtorrent.la/
```
Trailing space

File: definitions/v2/yggcookie.yml [hunk 10]
```diff
@@ -7,7 +7,7 @@ type: semi-private
 encoding: UTF-8
 followredirect: true
 links:
-  - https://www3.yggtorrent.re/
+  - https://www5.yggtorrent.la/
```
Comment (bakerboy448) [inline definitions/v2/yggcookie.yml:10]:
```suggestion
  - https://www5.yggtorrent.la/ 
```
Trailing space

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
Quick fixup

**Review by bakerboy448 — APPROVED** [review]

**Pull request merged**